src/bulk/common.ts

```diff
@@ -911,6 +911,8 @@ export class BulkWriteShimOperation extends AbstractOperation {
 /** @public */
 export abstract class BulkOperationBase {
   isOrdered: boolean;
+  // Declare dynamically assigned property
+  declare length: number;
```

Comment:
Can we declare the getter here?

```ts
  public get length(): number {
    return this.s.currentIndex;
  }
```

Reply:
I added the getter a little further down in the file, before the other getters.